neptune_optuna/impl/__init__.py

```diff
@@ -410,31 +420,32 @@ def log_study_metadata(
 
 
 def load_study_from_run(run: neptune.Run):
-    """A function that loads Optuna Study from an existing Neptune Run.
+    """Loads Optuna study from an existing Neptune run.
 
-    Loading mechanics depends on the study storage type used during the Neptune Run:
-    * if the study used 'InMemoryStorage', it will be loaded from the logged pickled Study object
-    * if the study used database storage, it will be loaded from the logged database URL
+    Loading mechanics depends on the study storage type used during the run:
+    - If the study used 'InMemoryStorage', it will be loaded from the logged pickled study object.
+    - If the study used database storage, it will be loaded from the logged database URL.
+    
+    To resume an existing run, you need the run ID. It is stored in the run's "sys/id" field.
 
     Args:
-        run(neptune.Run): Neptune Run.
+        run: Neptune run.
 
     Returns:
-        optuna.Study
+        Optuna study.
```

Comment:
this should be `optuna.Study`

Reply:
Wouldn't that just be the type? I'm intending to describe the returned object in plain language

Reply:
We generally mention the type..  Description can go along with it, like:
```
Returns:
    optuna.Study: <description>
````

Same with the args too. Although @Raalsky will be a better person to decide which docstring format we should follow

Mentioning types will also make the API reference more comprehensive if we decide on automating it in the new docs setup

Reply:
Also, another point to be added in the docs/code style guide would be a consistent docstring format across all our code

Reply:
We are actually several steps into this process already; per agreement with engineering, I've been driving the docstring standardization effort for some time. Some R&D has been necessary before settling on standards, but you're totally right that we need them written down ASAP. I've been playing around with Neptune code a bit more in these recent weeks and have some draft guidelines sketched out.

In essence: We are already using Google-style docstrings in many places, and I think that should be our main guideline going forward. Markup is non-existent (which should not cause rendering oddities in the various IDEs our users might be using) and stating argument types, defaults, etc. is left out if typing has been used in the definition, which seems to be the case at least in the public methods that I've come across.

(Addendum: after discussions especially with Mycek, API reference auto-generation is not something that will be considered in the foreseeable future.)

I'll likely be opening this topic up for review later this week.